client/blocks/inline-help/admin-sections.js

```diff
@@ -3,44 +3,49 @@
  */
 import { intersection, words, memoize } from 'lodash';
 import { translate } from 'i18n-calypso';
+import { getCustomizerUrl } from 'state/sites/selectors';
 
 /**
  * Internal Dependencies
  */
 import { getLocaleSlug } from 'lib/i18n-utils';
+import { SUPPORT_TYPE_ADMIN_SECTION } from './constants';
 
 /**
  * Returns admin section items with site-based urls.
  *
- * @param {string} siteSlug - The current site slug.
- * @returns {Array}           An array of admin sections with site-specific URLs.
+ * @param   {number} siteId   - The current site ID.
+ * @param   {string} siteSlug - The current site slug.
+ * @param   {object} state    - Global state
+ * @returns {Array}             An array of admin sections with site-specific URLs.
  */
-export const adminSections = memoize( ( siteSlug ) => [
+export const adminSections = memoize( ( siteId, siteSlug, state ) => [
 	{
 		title: translate( 'Add a new domain' ),
 		description: translate(
 			'Set up your domain whether it’s registered with WordPress.com or elsewhere.'
 		),
 		link: `/domains/add/${ siteSlug }`,
-		synonyms: [ 'domain' ],
+		synonyms: [ 'domains' ],
```

Comment:
Can we pass synonyms to translate() or should we wait for translations first?

Reply:
Not sure. Something to keep in mind in a next one? It's a good point.